rim/views.py

```diff
@@ -12,14 +13,36 @@
 from rim.models import Equipment, Checkout, EquipmentType, Location, Client
 from rim.forms import EquipmentForm
 
-class HomeView(ListView):
+class PaginateView(object):
```

Comment:
We should probably call this PaginateMixin instead (since it's not really a view)